hack/verify-codegen.sh

```diff
@@ -30,18 +35,24 @@ trap "cleanup" EXIT SIGINT
 
 cleanup
 
-# Save working tree state
 mkdir -p "${TMP_DIFFROOT}/pkg"
-cp -aR "${REPO_ROOT_DIR}/Gopkg.lock" "${REPO_ROOT_DIR}/vendor" "${TMP_DIFFROOT}"
+# Save working tree state
+for dir in ${DIRS_TOBE_INSPECTED[@]}; do
+  cp -aR "${REPO_ROOT_DIR}/${dir}" "${TMP_DIFFROOT}"
+done
 
 "${REPO_ROOT_DIR}/hack/update-codegen.sh"
 echo "Diffing ${REPO_ROOT_DIR} against freshly generated codegen"
 ret=0
-diff -Nupr --no-dereference "${REPO_ROOT_DIR}/vendor" "${TMP_DIFFROOT}/vendor" || ret=1
-diff -Nup --no-dereference "${REPO_ROOT_DIR}/Gopkg.lock" "${TMP_DIFFROOT}/Gopkg.lock" || ret=1
+for dir in ${DIRS_TOBE_INSPECTED[@]}; do
+  diff -Nupr --no-dereference "${REPO_ROOT_DIR}/${dir}" "${TMP_DIFFROOT}/${dir}" || \
+    { ret=1; echo "--- FAIL: file difference ${REPO_ROOT_DIR}/${dir}"; }
```

Comment:
Nit: would `${REPO_ROOT_DIR}/${dir} is not up-to-date`  be a better error message?